_posts/blog/2018-11-12-Latin@s-in-Ohio-review

```diff
@@ -0,0 +1,76 @@
+participants’ lived experiences and values, because to me it represented my families and our reality in the Midwest at a 
+specific moment in time. As a result, I felt connected to all of the individuals and proud to hear their stories. 
+           
+In addition, I was excited to find the majority of narratives were in Spanish, which was surprising due to the fact that 
+English is the dominant language in the Midwest. This important decision represented Dr. Foulis’ centering and celebration of 
+the Spanish language and Latinx culture, which was emphasized by the lack of English translations for the videos and shown by 
+the heritage speakers proudly using the language they are most comfortable with. This collection of video narrative is an 
+authentic example how the archive can bring to the forefront the voices of those usually excluded and marginalized to support 
+their efforts to preserve their stories.  
+
+Returning to my reactions as I read, I couldn’t help, but question my excitement. Why was I drawn to these stories, even 
+though I frequently and intentionally visited and engaged with Latinxs spaces? After reading “ To Suddenly Discover Yourself 
+Existing” by Caswell, et al., the aha moment appeared and they provided the words to describe my emotions. The authors 
+introduced symbolic annihilation and representational belonging and their meanings resonated with me. They defined symbolic 
+annihilation as the following “to be an eternal outsider whose very existence is presumed an impossibility”(Caswell et al. 
+58), and representational belonging “describes the affective responses community members have to seeing their communities 
+presented with complexity and nuance” (Caswell et al. 75). This resonated with me because I never realized that the stories of 
+my small, local marginalized community could be collected, seen as valued, and recorded for others to learn from our lived 
+experiences and for us to sustain our culture. This collection of video narratives Latinxs matter and we contribute valuable 
+perspective and knowledge to what it means to be Latinx in Ohio. 
+
+Nevertheless, I am left pondering what are the implications of this collection being held and supported by an institution of 
+higher education. As universities can be sites of oppression for marginalized groups. While reviewing the site and the 
+recordings, I was thrown off by the use of university propaganda as the background display. Was OSU trying to remind the 
+viewer they funded this project in order to engender positive perceptions of OSU? Equally important, is the use of written 
+English language to describe the project, why did Dr. Foulis and her team not include a Spanish version, for predominantly 
+Spanish speakers to read? With this in mind, I ask could this project be handed to community leaders to flourish into an 
+independent community archive, or what are the possibilities of community members can become collaborators of the project? 
```

Comment:
Grammar: this sentence has some grammatical errors. Perhaps a comma after "I ask" and "would it possible for community members to become collaborators..." would fix it.